packages/features/ee/teams/components/MakeTeamPrivateSwitch.tsx

```diff
@@ -46,6 +46,52 @@ const MakeTeamPrivateSwitch = ({
         switchContainerClassName="mt-6"
         data-testid="make-team-private-check"
       />
+      {/* TODO: add mutations and logic
+      {isOrg && (
+        <>
+          <SettingsToggle
+            toggleSwitchAtTheEnd={true}
+            title={t("hide_all_teams")}
+            disabled={disabled || mutation?.isPending}
+            description={t("hide_all_teams_description")}
+            checked={isTeamPrivate}
+            onCheckedChange={(checked) => {
+              setTeamPrivate(checked);
+              mutation.mutate({ id: teamId, isPrivate: checked });
```

Comment:
need help here. can be a follow up PR

Reply:
@PeerRich I think i already implemented hiding members for team. if you check the 'make team private' switch then you won't be able to see members

<img width="1512" alt="Screenshot 2024-02-22 at 1 26 35 AM" src="https://github.com/calcom/cal.com/assets/53316345/19d2d6d4-be37-4b49-9031-d89c7c0e166c">

Reply:
should i create separate switches for 'hiding event types', 'hiding members' and 'hiding team in orgs'